src/com/company/DecimalToBinary.java

```diff
@@ -0,0 +1,44 @@
+package com.company;
+
+/**
+ * An implementation of the decimal to binary conversion algorithm
+ */
+public class DecimalToBinary {
+
+    private static final int DECIMAL_NUMBER = -128;
+
+    private static String decimalToBinary(int decimalNumber) {
+
+        int binaryRepresentationOfANumber;
+
+        if (DECIMAL_NUMBER < 0) {
+            binaryRepresentationOfANumber = getBinaryRepresentationOfANumber(Math.abs(decimalNumber));
```

Comment:
We can make it easier to read and keep naming clear.

```suggestion
            binaryNumber = toBinary(Math.abs(decimalNumber));
```

Also, you can use `import static` for `Math.abs`